cmd/av/commit_amend.go

```diff
@@ -57,6 +60,13 @@ func runAmend(repo *git.Repo) error {
 		commitArgs = append(commitArgs, "--message", commitAmendFlags.Message)
 	}
 
+	tx := db.ReadTx()
+	branch, _ := tx.Branch(currentBranch)
+	// Todo: do we need to refresh the state of the branch
+	if branch.PullRequest != nil && branch.PullRequest.State == githubv4.PullRequestStateMerged {
```

Comment:
@draftcode am I correct in assuming we need to run `actions.SyncBranch` before checking this?

Reply:
actually just found `UpdatePullRequestState`